examples/slack_bot/requirements.txt

```diff
@@ -0,0 +1,3 @@
+embedchain
+embedchain[slack]
+embedchain[poe]
```

Comment:
Can we please pin the version of embedchain to use? You can also convert all of these in one line like this

```
embedchain[slack, poe]==0.1.7
```

Reply:
Sure, let me do that